addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -217,9 +221,11 @@
                                                 <input id="recruitment6" type="file"
                                                     class="form-control s_website_form_input o_resume_input"
                                                     name="Resume"/>
+                                                <span class="text-muted" style="font-size: 0.8rem">Optional if you provided a Linkedin profile</span>
                                             </div>
                                         </div>
-                                        <span class="text-muted" style="margin-left: 200px; font-size: 0.8rem">The resume is optional if you have a Linkedin profile</span>
+                                        <!-- TODO: remove in master -->
+                                        <span class="d-none text-muted" style="margin-left: 200px; font-size: 0.8rem">Optional if you provided a Linkedin profile</span>
```

Comment:
d-none is hiding it from all screens, just remove it. I don't get why do you change the text and then hide it

Reply:
Just remove the line :)

Reply:
> Just remove the line :)

We don't remove in stable version so the customization doesn't break.

Reply:
In this case, it's very unlikely :)